network/p2p/p2p_setup.go

```diff
@@ -161,15 +161,15 @@ func (n *p2pNetwork) setupStreamCtrl(logger *zap.Logger) error {
 }
 
 func (n *p2pNetwork) setupPeerServices(logger *zap.Logger) error {
-	libPrivKey, err := p2pcommons.ConvertToInterfacePrivkey(n.cfg.NetworkPrivateKey)
+	libPrivKey, err := p2pcommons.ECDSAPrivToInterface(n.cfg.NetworkPrivateKey)
 	if err != nil {
 		return err
 	}
 
 	domain := "0x" + hex.EncodeToString(n.cfg.Network.Domain[:])
 	self := records.NewNodeInfo(domain)
 	self.Metadata = &records.NodeMetadata{
-		OperatorID:  n.cfg.OperatorID,
+		OperatorID:  n.cfg.OperatorPublicKey,
```

Comment:
this is confusing ? is it operator ID or public Key?

Reply:
Well it seems like this is left overs from old code, in the metadata, OperatorID represents a hash of the operatorPublicKey ( you can see it on the comment where OperatorID is declared). the only thing that is changed here is the name of the value. I agree with @nkryuchkov that its better to save the change in `records.NodeMetadata` to a different PR that might require bigger refactoring and maybe a fork because of network message structure.